src/main/java/frc/robot/RobotContainer.java

```diff
@@ -22,80 +38,210 @@
 import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.button.CommandJoystick;
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import edu.wpi.first.wpilibj2.command.button.POVButton;
 import edu.wpi.first.wpilibj2.command.button.Trigger;
 
 /**
- * This class is where the bulk of the robot should be declared. Since Command-based is a
- * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
- * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
+ * This class is where the bulk of the robot should be declared. Since
+ * Command-based is a
+ * "declarative" paradigm, very little robot logic should actually be handled in
+ * the {@link Robot}
+ * periodic methods (other than the scheduler calls). Instead, the structure of
+ * the robot (including
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final MecanumDriveSubsystem m_Drive = new MecanumDriveSubsystem();
 
   private final LanuchMechanisumSubsystem m_LaunchMech = new LanuchMechanisumSubsystem();
-  private final IntakeMechanisum m_Intakemech = new IntakeMechanisum();
-  private final TrapScoreSubsystem m_TrapScoreSubsystem = new TrapScoreSubsystem();
+  private final LeftClimberSubsystem mLeftClimber = new LeftClimberSubsystem();
+  private final RightClimberSubsystem mRightClimber = new RightClimberSubsystem();
 
- // private Map<String, RelativeEncoder> Encoders = m_Drive.GetEncoders();
```

Comment:
I'll just include this comment once, but we should delete all of the code that has been commented out in the project.

Reply:
That took forever to do. Done though.